src/algebra/field.lean

```diff
@@ -46,8 +46,6 @@ def mk0 (a : α) (ha : a ≠ 0) : units α :=
 
 @[simp] theorem mk0_val (ha : a ≠ 0) : (mk0 a ha : α) = a := rfl
 
-@[simp] theorem mk0_inv (ha : a ≠ 0) : ((mk0 a ha)⁻¹ : α) = a⁻¹ := rfl
```

Comment:
Was it deleted for a reason?

Reply:
Yes, because it is exactly the same as `mk0_val`, but with a `⁻¹` around (the coercion is inserted inside the inverse).  It's not used anywhere, so I removed it.